src/views/level5/Level5-1.vue

```diff
@@ -113,35 +113,103 @@ export default {
   computed: {
     ...mapState('CommonsModule', ['minmax']),
     chart() {
-      const input = this.forms.input.decisions
+      const input = this.forms.input.decisions && this.forms.input.decisions > 2 ? this.forms.input.decisions : 3
+      const H = this.$store.state.CommonsModule.conviction.halflife
+      const H2 = this.$store.state.CommonsModule.conviction.inflectionPoint
+      const alpha = (1 - H) ** (1 / input)
+      const S = 1 - alpha
+      const t = Math.log((((alpha - 1) * H2) + S) / S) / Math.log(alpha)
+      const daysIndex = [3, 9, 15, 30, 60, 61].filter(x => x <= input).map((x, i) => i)
+      const maxBound = [45, 60, 180, 360, 360, 720][daysIndex.pop()]
```

Comment:
I would set a fixed x axis of 180 days to make sure everyone understand the plot at the first glance.

It's just changing these two lines by `const maxBound = 180`.